api/src/main/java/io/druid/data/input/impl/DimensionSchema.java

```diff
@@ -26,10 +26,12 @@
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
 import com.fasterxml.jackson.annotation.JsonValue;
 import com.google.common.base.Preconditions;
+import io.druid.guice.annotations.PublicApi;
 import io.druid.java.util.common.StringUtils;
 
 /**
  */
+@PublicApi
```

Comment:
is this public api so that users can use this class in their java clients to construct druid queries ?